docs/hyperloop/userdocumentation.md

```diff
@@ -269,7 +269,81 @@ When creating or enabling wagons, you can use a pull request instead of a packag
+ * For the other target queues, trains can only be composed if the memory consumption is within the target limits.
+ * For the cases when the train cannot be composed due to high memory consumption, the user can review the test. One can check the logs and look for any possible improvements that can be done for a lower memory consumption.
+ 
+ ### 2. <a name="warning-pss"></a> Maximal PSS more than 30% larger than average PSS
+ 
+  <div align="center">
+    <img src="../images/warningPSS.png" width="40%">
+  </div>
+ 
+ * The maximum PSS memory consumption is larger than 30% of the average PSS, therefore the train cannot be automatically composed. The test will be checked by the operator and, if there is no memory leak, the train can be composed. Otherwise, they will advise the user to check for possible causes and improvements before requesting again.
 
+ ### 3. <a name="warning-cpu"></a> CPU usage too large
+ 
+  <div align="center">
+    <img src="../images/warningCPU.png" width="40%">
+  </div>
+ 
+ * The CPU usage limit is set per dataset and all trains running on a specific dataset must respect this constraint. If the limit is not respected, the train cannot be composed without PWG approval. Therefore, the user should discuss the details and requirements for this train with the PWG before requesting again.
+ 
+ ### 4. <a name="warning-ccdb"></a> Too many CCDB calls
+ 
+  <div align="center">
+    <img src="../images/warningCCDB.png" width="40%">
+  </div>
+ 
+ * Too many calls to the CCDB, therefore the train cannot be composed, and the cause of a high number of calls should be checked.
+
+ ### 5. <a name="warning-reduction-factor"></a> Reduction factor too small
+ 
+  <div align="center">
+    <img src="../images/warningReductionFactor.png" width="40%">
+  </div>
+ 
+ * This occurs when the reduction factor is lower than 50. If the expected output size is below 10 GB, the operator can compose the train on request. If larger, the train cannot be composed.
+ 
+ ### 6. <a name="warning-log-file"></a> Log output too large
+ 
+  <div align="center">
+    <img src="../images/warningLogOutput.png" width="40%">
+  </div>
+ 
+ * The log file is too large, therefore the train cannot be composed, and the user should check for factors leading to this.
+ 
+ ### 7. <a name="warning-derived-output"></a> Derived output too large for slim train
+ 
+  <div align="center">
+    <img src="../images/warningDerivedOutput.png" width="40%">
+  </div>
+ 
+ * This is specific to tests with wagons set as ready for slim derived data. It can happen when the estimate is slightly different during the wagon test and the train test. If the output is lower than 5000 MB, the operator can submit the train on request. 
+ * Otherwise, the user is advised to switch to standard derived data by unchecking the option “Ready for slim derived data” in the wagon edit view. Then a request for standard derived data train can be made.
```

Comment:
```suggestion
 * This is specific to tests with wagons set as ready for slim derived data. As the entire output is merged into one single file, there is a limit of 4000 MB for. The user is advised to switch to standard derived data by unchecking the option “Ready for slim derived data” in the wagon edit view. Then a request for standard derived data train can be made.
```
(I think we should focus here on the user view, not what the operator would do if it appears in the train test)